package.json

```diff
@@ -6,14 +6,15 @@
     "serve:dev-agent-1": "VUE_APP_WEB_CLIENT_PORT=8888 yarn serve:develop",
     "serve:dev-agent-2": "VUE_APP_WEB_CLIENT_PORT=9999 yarn serve:develop",
     "serve:develop": "VUE_APP_UI_VERSION=$npm_package_version VUE_APP_CONTEXT='holochain' vue-cli-service serve",
-    "serve": "VUE_APP_UI_VERSION=$npm_package_version VUE_APP_CONTEXT='holo-host' vue-cli-service serve",
-    "build": "VUE_APP_UI_VERSION=$npm_package_version VUE_APP_CONTEXT='holo-host' vue-cli-service build",
+    "serve": "VUE_APP_UI_VERSION=$npm_package_version VUE_APP_CONTEXT='holo-host' VUE_APP_CHAPERONE_SERVER_URL='http://localhost:24273/' vue-cli-service serve",
+    "build": "VUE_APP_UI_VERSION=$npm_package_version VUE_APP_CONTEXT='holo-host' VUE_APP_CHAPERONE_SERVER_URL='https://chaperone.holo.host/' vue-cli-service build",
     "test:unit": "vue-cli-service test:unit",
     "test:e2e": "vue-cli-service test:e2e",
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "@holochain/conductor-api": "git://github.com/holochain/holochain-conductor-api.git#2020-12-10/timeout",
+    "@holo-host/web-sdk": "^0.2.2",
+    "@holochain/conductor-api": "git://github.com/holochain/holochain-conductor-api.git#b6cb529ce0e446dd172e120b27b90d104aa7369b",
```

Comment:
`2020-12-10/timeout` branch of `conductor-api` was deleted. This is the commit it was pointing to when this version was installed.

Reply:
v0.0.1-dev.14 is the released version with the set-timeout fixes.  See: https://github.com/holochain/holochain-conductor-api/pull/41